src/index.ts

```diff
@@ -36,19 +37,76 @@ export {HttpRequest};
 export {detectServiceContext};
 
 const PKG = require('../../package.json');
-const v2 = require('./v2');
+import * as v2 from './v2';
 
-import {Entry} from './entry';
-import {Log, GetEntriesRequest, MonitoredResource, Severity, SeverityNames} from './log';
+import {Entry, LogEntry} from './entry';
+import {Log, GetEntriesRequest, LogOptions, MonitoredResource, Severity, SeverityNames} from './log';
 import {Sink} from './sink';
 import {Duplex} from 'stream';
 import {AbortableDuplex} from '@google-cloud/common';
+import {google} from '../proto/logging';
+import {google as google_config} from '../proto/logging_config';
+import {CallOptions} from 'google-gax/build/src/gax';
+import {Bucket} from '@google-cloud/storage';
```

Comment:
Are these imported for Types only, or do we actually use the modules?  If types only, please add `// types only` after.  I'm a little worried about the weight of these imports.

Reply:
These are for types only